core/src/parameter.jl

```diff
@@ -1,3 +1,6 @@
+const LOW_STORAGE_THRESHOLD = 10.0
```

Comment:
Please comment on why/where this is needed. Like `# below this storage threshold the reduction factor is triggered`.